data/core/common_data.sh

```diff
@@ -61,6 +61,20 @@ initialize_package_manager_pkg() {
   PACKAGE_MANAGER_AUTOCLEAN="pkg -y autoclean"
 }
 
+initialize_package_manager_pacman() {
+  DEFAULT_PACKAGE_MANAGER="pacman"
+  PACKAGE_MANAGER_INSTALL="pacman -S"
```

Comment:
Why not assuming yes when installing a package? user interaction requests are forbidden in customizer